adapters/fiducia-adapter/src/main/java/de/adorsys/xs2a/adapter/fiducia/FiduciaAccountInformationService.java

```diff
@@ -36,9 +36,10 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+import java.util.Arrays;
 
 public class FiduciaAccountInformationService extends BaseAccountInformationService {
-    private static final Set<String> SUPPORTED_BOOKING_STATUSES = new HashSet<>(Collections.singletonList("booked"));
+    private static final Set<String> SUPPORTED_BOOKING_STATUSES = new HashSet<>(Arrays.asList("booked", "information"));
```

Comment:
@jkaldzija Thanks for your contribution, for the new code `Set.of` should be used as it provides out-of-the-box immutability 
`private static final Set<String> SUPPORTED_BOOKING_STATUSES = Set.of("booked", "information")`

Also, please add the new test case for the new `SUPPORTED_BOOKING_STATUSES` to FiduciaAccountInformationServiceTest.java.